addons/project/static/src/js/project_calendar.js

```diff
@@ -22,10 +23,23 @@ const ProjectCalendarController = CalendarController.extend({
     },
 });
 
+const ProjectCalendarModel = CalendarModel.extend({
+    /**
+     * @private
+     * @override
+     */
+    _getFullCalendarOptions: function () {
+        const options = this._super(...arguments);
+        options.eventDurationEditable = false;
```

Comment:
I don't find any occurrence of that option 🤔 
Can you explain what this option does? 🙂

Reply:
It allows resizing events to change their duration: https://fullcalendar.io/docs/eventDurationEditable

Reply:
Okay thanks. 🙂